src/main/plugins/org.dita.pdf2/xsl/fo/index.xsl

```diff
@@ -492,7 +492,7 @@ See the accompanying LICENSE file for applicable license.
     <xsl:param name="refID"/>
 
     <xsl:for-each select="$index-entries">
-      <xsl:variable name="entries" select="key('index-key',$value)" as="element()*"/>
+      <xsl:variable name="entries" select="key('index-key',$value)[1]" as="element()*"/>
```

Comment:
Is it correct to only take the first entry here? In the next line we filter based on the ID and `value-of` will handle outputting only the first one.

Reply:
You are correct--should not have a [1] there. Missed the element()*

Reply:
That code still looks a little iffy on its face--doesn't look good to potentially get multiple elements and then apply value-of to them but I don't know what this code is really doing or why so will simply remove the [1]